client/jetpack-onboarding/constants.js

```diff
@@ -1,5 +1,12 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import i18n from 'i18n-calypso';
```

Comment:
Let's import only `translate` and call `translate()` instead of `i18n.translate()` down there.